### PR TITLE
Add information about multiple option

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

**Comment by joneslee85** [comment]
:+1:

File: lib/hanami/helpers/form_helper/form_builder.rb [hunk 1]
```diff
@@ -486,6 +486,7 @@ def hidden_field(name, attributes = {})
         # @param name [Symbol] the input name
         # @param attributes [Hash] HTML attributes to pass to the input tag
         # @option attributes [String,Array] :accept Optional set of accepted MIME Types
+        # @option attributes [TrueClass,FalseClass] :multiple Optional, allow multiple file upload
```
Comment (joneslee85) [inline lib/hanami/helpers/form_helper/form_builder.rb]: what would be the output if user provides `false`?

Comment (davydovanton) [inline lib/hanami/helpers/form_helper/form_builder.rb]: this method will return `<input>` tag without `multiple` option

Comment (joneslee85) [inline lib/hanami/helpers/form_helper/form_builder.rb]: is there test for this option?

Comment (davydovanton) [inline lib/hanami/helpers/form_helper/form_builder.rb]: [yep](https://github.com/hanami/helpers/blob/master/test/form_helper_test.rb#L661-L667)

**Comment by davydovanton** [comment]
thanks :+1:

**Comment by jodosha** [comment]
@davydovanton @joneslee85 Thank you :+1:

**Pull request merged**